tsctl

```diff
@@ -134,7 +126,7 @@ class GroupManager(Command):
         super(GroupManager, self).__init__()
 
     # pylint: disable=arguments-differ, method-hidden
-    def run(self, add, remove, group_name, user_name):
+    def run(self, remove, group_name, user_name):
         """Add the user to the group."""
```

Comment:
maybe add a docstring here

Reply:
Yeah, tsctl is a weird leftover from early days. These arguments are passed down to the Command class from the flask-script package. Flaks-script have actually been deprecated so we need to port it to flask.cli anyways. This will be done in a followup PR though as this is a bit involved.